vk.py

```diff
@@ -0,0 +1,44 @@
+import os
+import json
+import re
+import urllib
+import constants
+
+
+def vk_get():
+    # get data from vk.com
+    vk_id = constants.vk_id
+    adr = 'https://api.vk.com/method/wall.get?owner_id={0}&count=1'.format(vk_id)
+    data = urllib.request.urlopen(adr)
+    decoded_response = data.read().decode()
+    final_data = json.loads(decoded_response)
+    # getting last post response
+    response = final_data['response'][1:2]
+    for resp in response:
+        current_id = resp['id']  # getting an id
+        text = resp['text']  # getting txt
+    # getting previous id and writing new id
+    if os.path.exists('last_id.txt'):
+        with open('last_id.txt') as f:
+            prev_id = f.read()
+    else:
+        prev_id = '0'
```

Comment:
Again - do you read what I'm telling to you? I've written about this three days ago!!! :angry: